Makefile.am

```diff
@@ -263,6 +285,7 @@ distclean-local:
 	rm -f libcoap-$(LIBCOAP_API_VERSION).map
 	rm -f libcoap-$(LIBCOAP_API_VERSION).sym
 	rm -f libcoap-$(LIBCOAP_NAME_SUFFIX).pc
+	rm -f libcoap.pc
```

Comment:
Where would this file come from? The install-exec-hook creates `libcoap-$(LIBCOAP_API_VERSION).pc`.

Reply:
Mental aberration - good catch - it should not be there at all as the` libcoap-$(LIBCOAP_API_VERSION).pc` file is only ever created in ` $(pkgconfigdir).`

Errant update backed out and code pushed.